tests/models/roc_bert/test_modeling_roc_bert.py

```diff
@@ -705,4 +705,4 @@ def test_inference_masked_lm(self):
         # convert to tokens is: ['[CLS]', '巴', '*', '黎', '是', '法', '国', '的', '首', '都', '[SEP]']
         expected_output = torch.tensor([[101, 2349, 115, 7944, 3221, 3791, 1744, 4638, 7674, 6963, 102]])
 
-        self.assertTrue(output_ids, expected_output)
+        self.assertTrue(torch.allclose(output_ids, expected_output))
```

Comment:
```suggestion
        self.assertEqual(output_ids, expected_output)
```

Reply:
So younes was right : 
```python
>>> self.assertEqual(output_ids, expected_output)
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/opt/conda/envs/py39/lib/python3.9/unittest/case.py", line 837, in assertEqual
    assertion_func(first, second, msg=msg)
  File "/opt/conda/envs/py39/lib/python3.9/unittest/case.py", line 827, in _baseAssertEqual
    if not first == second:
RuntimeError: Boolean value of Tensor with more than one value is ambiguous
```

Opening up a [PR to fix this ](https://github.com/huggingface/transformers/pull/20469)